src/table/styles.scss

```diff
@@ -68,7 +68,8 @@
   scrollbar-width: none; /* Hide scrollbar in Firefox */
   &.variant-stacked,
   &.variant-container {
```

Comment:
What happens in the full page variant? If all works as expected, should this be reflected in the tests?

Reply:
It works just fine because there are no extra paddings. Added a test for this variant as well.